lib/maths/analytics/unittest/CBoostedTreeTest.cc

```diff
@@ -3776,11 +3776,11 @@ BOOST_AUTO_TEST_CASE(testStopAfterCoarseParameterTuning) {
                 1, std::make_unique<maths::analytics::boosted_tree::CMse>())
                 .numberHoldoutRows(numberHoldoutRows)
                 .buildForTrain(*frame, cols - 1);
-        return regression->hyperparameters().optimisationMakingNoProgress();
+        return regression->hyperparameters().fineTuneSearchNotFinished();
     };
 
-    BOOST_REQUIRE_EQUAL(verify(0.001), false);
-    BOOST_REQUIRE_EQUAL(verify(1000.0), true);
+    BOOST_REQUIRE_EQUAL(verify(0.0), true);
+    BOOST_REQUIRE_EQUAL(verify(1000.0), false);
```

Comment:
Why are the check values swapped now?

Reply:
This is just because I swapped it to use `fineTuneSearchNotFinished()` which is the negation of what it was before. So this really means the same thing, i.e. true means we're going to carry on training and false means we'll stop.